crates/sui-cost/data/estimate.toml

```diff
@@ -0,0 +1,44 @@
+[MergeCoin]
+computation_cost = 468
+storage_cost = 32
+storage_rebate = 0
+
+[Publish]
+computation_cost = 526
+storage_cost = 83
+storage_rebate = 0
+
+[SharedCounterAssertValue]
+computation_cost = 198
+storage_cost = 31
+storage_rebate = 15
```

Comment:
probably noob q: why does this  and `SharedCounterIncrement` have storage rebate?

Reply:
great question. Need to look into it myself